bin/s2nd.c

```diff
@@ -303,6 +303,8 @@ void usage()
     fprintf(stderr, "  -n\n");
     fprintf(stderr, "  --negotiate\n");
     fprintf(stderr, "    Only perform tls handshake and then shutdown the connection\n");
+    fprintf(stderr, "  --parallelize\n");
+    fprintf(stderr, "    Create a new Connection handler thread for each new connection. Useful for tests with lots of connections.\n");
```

Comment:
Should we just make this the default?

Reply:
I originally did, but ended up making it optional because it breaks the TLS Resumption tests since the `s2nd` session cache is not shared between threads.

Reply:
Ah that makes sense. I think we can move it to the default in a later PR.